packages/tasks/src/model-libraries-snippets.ts

```diff
@@ -82,6 +82,53 @@ export const bm25s = (model: ModelData): string[] => [
 retriever = BM25HF.load_from_hub("${model.id}")`,
 ];
 
+export const depth_anything_v2 = (model: ModelData): string[] => {
+	let encoder: string;
+	let features: string;
+	let out_channels: string;
+
+	encoder = "<ENCODER>";
+	features = "<NUMBER_OF_FEATURES>";
+	out_channels = "<OUT_CHANNELS>";
+
+	if (model.id === "depth-anything/Depth-Anything-V2-Small") {
+		encoder = "vits";
+		features = "64";
+		out_channels = "[48, 96, 192, 384]";
+	} else if (model.id === "depth-anything/Depth-Anything-V2-Base") {
+		encoder = "vitb";
+		features = "128";
+		out_channels = "[96, 192, 384, 768]";
+	} else if (model.id === "depth-anything/Depth-Anything-V2-Large") {
+		encoder = "vitl";
+		features = "256";
+		out_channels = "[256, 512, 1024, 1024";
+	}
```

Comment:
I think using a switch or something like this might be cleaner, but I'm not a typescript developer so maybe someone from the Hub team can provide guidance. Otherwise feel free to go ahead with current code

```suggestion
	const modelConfigs = {
          "depth-anything/Depth-Anything-V2-Small": {
            encoder: "vits",
            features: "64",
            out_channels: "[48, 96, 192, 384]"
          },
          "depth-anything/Depth-Anything-V2-Base": {
            encoder: "vitb",
            features: "128",
            out_channels: "[96, 192, 384, 768]"
          },
          "depth-anything/Depth-Anything-V2-Large": {
            encoder: "vitl",
            features: "256",
            out_channels: "[256, 512, 1024, 1024]"
          }
        };

	let config = modelConfigs[model.id] || {
          encoder: "<ENCODER>",
          features: "<NUMBER_OF_FEATURES>",
          out_channels: "<OUT_CHANNELS>"
        };
        
        const { encoder, features, out_channels } = config;
```